tests/saitests/py3/sai_qos_tests.py

```diff
@@ -1024,18 +1024,17 @@ def runTest(self):
                 print(list(map(operator.sub, pg_cntrs, pg_cntrs_base)),
                       file=sys.stderr)
                 for i in range(0, PG_NUM):
+                    # In DNX, ip2me packets trapped to cpu mapped to TC 1 -> PG 0 and
+                    # all other control packets trapped to cpu mapped to TC 4 -> PG 4.
                     if i == pg:
-                        if i == 0 or i == 4:
-                            assert (pg_cntrs[pg] >=
-                                    pg_cntrs_base[pg] + len(dscps))
+                        if i == 0 or i == 4 or i == 7:
```

Comment:
I see this code as a common sai qos test library - do we need to add a check for broadcom DNX ASIC  ? else this will get applicable to all asic platforms

Reply:
@judyjoseph , added the broadcom DNX ASIC check. Kindly review.

Reply:
thanks. My original comment was to protect just this new change i.e i=7 alone with the check for asic == dnx. The other checks for i=0, i=4 was existing earlier, will leave it as such